sysfs/class_nvme.go

```diff
@@ -45,7 +45,7 @@ func (fs FS) NVMeClass() (NVMeClass, error) {
 
 	dirs, err := ioutil.ReadDir(path)
 	if err != nil {
-		return nil, fmt.Errorf("failed to list NVMe devices at %q: %v", path, err)
+		return nil, err
```

Comment:
Thanks, good catch!
But I think we should do this instead:
```suggestion
		return nil, fmt.Errorf("failed to list NVMe devices at %q: %w, path, err)
```
That should also allow checking for the error type: https://blog.golang.org/go1.13-errors

Reply:
Thanks for the pointer to the documentation. That helps a lot since I am still quite new to Go.

Changed and adjusted commit message and merge request.